setup.py

```diff
@@ -51,6 +51,8 @@
     'sendgrid==5.6.0',
     'simple-crypt==4.1.7',
     'unicodecsv==0.14.1',
+    # Profiler related.
+    'google-cloud-profiler==1.0.8',
```

Comment:
Apologies, I just recalled that @ahoying mentioned that instrumentation packages to be optional, so that it's easier for him to port into Google's environment.  @ahoying, please confirm?

If so, the optional dependency can be specified like this in `setup.py`:
![image](https://user-images.githubusercontent.com/6977544/63400580-2c23f280-c389-11e9-8c8c-8dbc70e8c852.png)

Then, where you use the import in `server.py`, you can do this try/except:
![image](https://user-images.githubusercontent.com/6977544/63400625-583f7380-c389-11e9-8f71-e6c060dd5fcf.png)

Then when we want to use it, we can do this:
`pip3 install .[profiler]`

I took the screenshots from this PR #2317, where this optionality is implemented, so we should do the same here with the profiler.